src/worker-tests/worker.test.ts

```diff
@@ -24,19 +24,16 @@ test.skip('worker test', async () => {
   dbsToDrop.add(name);
 
   {
-    const data = await send(w, {name, useMemstore: false});
+    const data = await send(w, {name});
```

Comment:
seems we can delete one of these blocks as they are now redundant

Reply:
Thanks